sips/fork_support.md

```diff
@@ -20,56 +20,204 @@ Client implementation should calculate when a new duty starts to decide if it st
 **History Sync**  
 Sycing messages from a peer stays unchanged as messages have the identifier in them with domain type 
 
-[Suggested changes to SSV-Spec](https://github.com/bloxapp/ssv-spec/compare/main...alonmuroch:ssv-spec:ssv-fork-updated)
 
-**Spec Changes**
+## Current State
 
-type changes
+Currently, there's **no fork change logic**. However, there's **network and fork identification** through the `DomainType`. For instance, we have
 ```go
-// DomainType is a unique identifier for signatures, 2 identical pieces of data signed with different domains will result in different sigs
 type DomainType [4]byte
+var (
+	GenesisMainnet = DomainType{0x0, 0x0, 0x0, 0x0}
+	PrimusTestnet  = DomainType{0x0, 0x0, 0x1, 0x0}
+	ShifuTestnet   = DomainType{0x0, 0x0, 0x2, 0x0}
+	ShifuV2Testnet = DomainType{0x0, 0x0, 0x2, 0x1}
+	V3Testnet      = DomainType{0x0, 0x0, 0x3, 0x1}
+)
+```
+The _DomainType_ major importance is the fact that **it's mixed in the signature** such that messages on different domains will have different signatures.
+
+The node takes one of the above _DomainType_ values and **holds it until termination**.
```

Comment:
I think now this sentence is confusing

Reply:
Do you mean the "The node takes one of the above _DomainType_ ..." sentence? Notice that it describes the previous state

Reply:
ahh ok, my bad